Gruntfile.js

```diff
@@ -0,0 +1,133 @@
+module.exports = function (grunt) {
+    'use strict';
+
+    var _ = require('lodash');
+    var fs = require('fs');
+
+    var projectTemplates = {};
+
+    var modules_dir = 'modules/'; // eslint-disable-line camelcase
```

Comment:
Instead of disabling linting for this thing, just rename it to `modulesDir` so it matches our styleguide and the rest of the file in being `camelCase`